pandas/io/parsers.py

```diff
@@ -227,6 +227,21 @@
     Note that the entire file is read into a single DataFrame regardless,
     use the `chunksize` or `iterator` parameter to return the data in chunks.
     (Only valid with C parser)
+compact_ints : boolean, default False
+    DEPRECATED: this argument will be removed in a future version
+
+    If compact_ints is True, then for any column that is of integer dtype,
+    the parser will attempt to cast it as the smallest integer dtype possible,
+    either signed or unsigned depending on the specification from the
+    `use_unsigned` parameter. Selecting this option can help save memory if
+    there are a lot of relatively small integers in the resulting DataFrame.
```

Comment:
Hmmm...that is a fair point.  Perhaps I'll just take that out then because trying to clarify the correct meaning would make things overly complicated I'd imagine.